### ISSUE TITLE
Keyboard navigation (a11y)

### ISSUE THREAD

**Issue by a0m0rajab** [calcom/cal.com#8887]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

Being on the availability page, when opening the copy to menu, it would not allow to naviage with keyboard. 
https://app.cal.com/availability

Plus, I think the accessibility could be improved. 

### Steps to Reproduce

1. go to: https://app.cal.com/availability
2. Chose a schedule
3. open copy to
4. try to use ONLY your keyboard to enable a button. 

The copy to menu is not navigatable through keyboard: 
![image](https://github.com/calcom/cal.com/assets/18273833/59d9c60d-65b8-425e-bafe-5dc3ae9422cb)

### suggestions

I tried to solve this with tabindex but did not work, here are a few links that I looked for to solve the issue but could not: 
- https://web.dev/control-focus-with-tabindex/
- https://stackoverflow.com/questions/52998354/why-does-tabindex-not-work-after-first-tab-to-select-li
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard

**Comment by abdullahkhan70** [comment]
@a0m0rajab there is no copy option. Can you show it through some screenshots? In this way, I can become easy to understands the issue.

**Comment by a0m0rajab** [comment]
Added an image and changed the description, thank you.

**Comment by a0m0rajab** [comment]
We have the same issue when the tips are open: 
![tips and profile](https://github.com/calcom/cal.com/assets/18273833/771b5eb8-e170-4829-8929-5921fce0c42e)

the keyboard goes to the last card in the tips then it got to the first one, I think it should show the first one the go to the profile. 

right now, it's  skipping the profile.

**Comment by SksOp** [comment]
@PeerRich I would like to inquire about the expected behavior of a keyboard in two different cases. Please find the details below:

**Case 1:**

- The first checkbox should be automatically focused when the interface is loaded.

- Users should be able to navigate through the checkboxes using the arrow keys.

**Case 2:**

- Upon pressing the Tab key or arrow key, the focus should be directed to the first checkbox.

- Once the focus is on the first checkbox, users should be able to navigate through the checkboxes using the arrow keys.

The user can able to toggle the checkbox using Enter key in both the cases.

Please tell me which case should I implement in the code.
Thanks

**Comment by a0m0rajab** [comment]
@SksOp in either solution have you tried to check them in the code? 
cause I was expecting to be able to navigate with arrow keys, but I could not use any of them, in the dev env.

**Comment by SksOp** [comment]
Yes, I was able to implement the feature in the code to get the navigation throughout the checkboxes using keyboard by pressing Tab key. But I thought some one might try to itterate using navigation key also. So I was wondering which one should I implement.

**Comment by a0m0rajab** [comment]
That's great! if it does not take much effort from your side, did you think to open two PRs and let peer decide on which one they prefer? (this might save time from the conversation going back and forth) 

And I got curios how you implemented it.

**Comment by a0m0rajab** [comment]
@SksOp would not this be done with HTML or CSS without the need for extra logic?

**Comment by SksOp** [comment]
To do that using HTML , I guess we might require to use <form> wrapping this over form will do I guess, but its a theory. I haven't tried it yet
